### PR TITLE
new icon: Figma (original)

### PR DESCRIPTION
**Pull request by AanchalCh**
Original figma icon in svg format and eps format added #202

### CODE REVIEW

File: icons/figma/figma-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 300"><style>.st0{fill:#0acf83}.st1{fill:#a259ff}.st2{fill:#f24e1e}.st3{fill:#ff7262}.st4{fill:#1abcfe}</style><path id="path0_fill" class="st0" d="M50 300c27.6 0 50-22.4 50-50v-50H50c-27.6 0-50 22.4-50 50s22.4 50 50 50z"/><path id="path1_fill" class="st1" d="M0 150c0-27.6 22.4-50 50-50h50v100H50c-27.6 0-50-22.4-50-50z"/><path id="path1_fill_1_" class="st2" d="M0 50C0 22.4 22.4 0 50 0h50v100H50C22.4 100 0 77.6 0 50z"/><path id="path2_fill" class="st3" d="M100 0h50c27.6 0 50 22.4 50 50s-22.4 50-50 50h-50V0z"/><path id="path3_fill" class="st4" d="M200 150c0 27.6-22.4 50-50 50s-50-22.4-50-50 22.4-50 50-50 50 22.4 50 50z"/></svg>
```
Comment (amacado) [inline icons/figma/figma-original.svg:1]: [CONTRIBUTING.md#organizational-guidelines](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#organizational-guidelines): .svg file contains one version of an icon in a "0 0 128 128" viewbox

File: icons/gatsby/gatsby-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128"><style type="text/css">.st0{fill:#663399;}.st1{fill:#FFFFFF;}</style><g><circle class="st0" cx="14" cy="14" r="14"/><g><path class="st1" d="M6.2,21.8C4.1,19.7,3,16.9,3,14.2L13.9,25C11.1,24.9,8.3,23.9,6.2,21.8z"/><path class="st1" d="M16.4,24.7L3.3,11.6C4.4,6.7,8.8,3,14,3c3.7,0,6.9,1.8,8.9,4.5l-1.5,1.3C19.7,6.5,17,5,14,5c-3.9,0-7.2,2.5-8.5,6L17,22.5c2.9-1,5.1-3.5,5.8-6.5H18v-2h7C25,19.2,21.3,23.6,16.4,24.7z"/></g></g></svg>
```
Comment (amacado) [inline icons/gatsby/gatsby-original.svg:1]: [CONTRIBUTING.md#organizational-guidelines](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#organizational-guidelines): .svg file contains one version of an icon in a "0 0 128 128" viewbox

File: icons/figma/figma-original.svg (hunk 1 above)
Comment (AanchalCh) [reply icons/figma/figma-original.svg:1]: screencapture-file-C-Users-HP-Documents-Github-devicon-icons-figma-figma-original-svg-2020-07-29-16_48_22

This is how my Chrome window (1440x1080) is displaying the Figma svg and if I set the viewBox to 0 0 128 128 it's getting cropped out and not displaing correctly. I can't figure out why. I tried to do `preserveAspectRatio="xMidYMid meet"` in the second case but that didn't change anything.

screencapture-file-C-Users-HP-Documents-Github-devicon-icons-figma-figma-original-svg-2020-07-29-16_55_14 (1)

**Comment by Thomas-Boi** [comment]
Hello there,
Recently, we just introduced an automation pipeline to Devicon which will help us create the icons. Before we can merge your changes in, there are a few things that we need to do first:

1. Merge the current devicon/develop branch into your PR branch. This contains the build script needed for us to build your icons.
2. Read the `CONTRIBUTING.md` for things to do before merging and review our icon standards.
3. Push your changes and/or branches. We will review them once this is done and merge your PR if everything goes well.


Thank you for contributing to this repo. We are sorry that we haven't accept any PR for awhile. Now that we have a build pipeline, we can merge PRs more quickly than before. 

We hope to see your icons in the Devicon repo soon :)

Cheers,
Devicon Team

**Comment by Thomas-Boi** [comment]
Hello there,

We notice that this PR has been inactive for awhile now. It would be a waste of your time and effort if we just close this PR and throw away the SVGs in here. Thus, 1 month from now, if this PR remains inactive/stale, we will fork your repo, fix any issues, and merge your icons ourselves.

Since GitHub tracks contributions by commits, you will still be credited. We also hope that you will help us merge this PR before the deadline comes.

Cheers,
Devicon Team

**Comment by amacado** [comment]
Figma was added in https://github.com/devicons/devicon/pull/531

**Pull request closed without merging**